src/routes/__tests__/express.test.js

```diff
@@ -30,6 +28,41 @@ describe('Basic API', () => {
         });
     });
 
+    it('returns sharing disabled for config if no secrets specified', async () => {
+        const nodule = Nodule.testing();
+        await nodule.load();
+
+        const { express, config } = getContainer('routes');
+        express.get('/api/config', config);
+
+        const response = await request(express).get(
+            '/api/config',
+        );
+
+        expect(response.statusCode).toBe(200);
+        expect(response.body).toEqual(
+            'Config sharing disabled if no secrets are labeled',
+        );
+    });
+
+    it('returns config if secrets are specified', async () => {
+        const nodule = Nodule.testing();
+        await nodule.load();
+        bind('config.secrets', () => ['middleware']);
```

Comment:
We probably want to obscure specific keys, not just blanket keys (e.g. the client ID/audience may be okay to expose here, just not the middleware secret). I think that's how we make it work in microcosm (also, it looks like we only remove the keys associated with the secrets?)

Reply:
Hm, never mind, I see this is just covering over the `secrets` key from `secretsmanager` in `nodule-config`.